src/dbus_proxy/_proxy.lua

```diff
@@ -175,6 +192,45 @@ local function call(proxy, interface, method, args)
   return result
 end
 
+--- Asynchronously call a method with arguments from a given interface on a proxy object.
```

Comment:
Can you add a reference to the GLib docs please?

- https://developer.gnome.org/gio/stable/GDBusProxy.html#g-dbus-proxy-call
- https://developer.gnome.org/gio/stable/GDBusProxy.html#g-dbus-proxy-call-finish